### DISCUSSION TITLE
Move Uyuni Community Hours to Thursdays

### DISCUSSION THREAD

**Discussion by raulillo82** [uyuni-project/uyuni#9302] [Polls]
I would like to bring up the topic of the Uyuni community hours on Fridays.

- For many/some people (the ones in Europe) it is the last hour of the last working day. Some might be already off
- Some workplaces have a no-meetings Friday policy
- Feedback in events about European places not even working on Fridays after 3pm CEST (one hour before the meeting).
- It's a day more prone to have vacation than a day in the middle of the weeks.
- Some conflicts for moving the meeting to Thursdays disappear and this is now a valid option, at least for the bigger part of the audience, SUSE engineers.
- We decided to suggest Thursdays at 4pm CET/CEST (or right now 2pm UTC), to keep it reasonable for the colleagues/community members on the other side of the ocean.
- We will keep the last week of the month, as usual

I'm starting this poll and giving a deadline to vote till the end of Thursday, October 24th. If the poll has no big remarks against, then the next session of Uyuni Community hours would take place on October 31st (there would be a separate announcement for this, probably via email).

Please also feel free to add any comments/feedback, this is a discussion.

**Comment by deneb-alpha** [comment]
The majority of the voters decided to move the community hours to the last Thursdays of the month at 4pm CEST/CET. 

Closing the poll
